app/models/profile.rb

```diff
@@ -0,0 +1,5 @@
+class Profile < ApplicationRecord
+    belongs_to   :customer
+
+    enum sex: { 男: 0, 女: 1 }
```

Comment:
[must]
enumの定義に日本語は使わない方がいいですね👾
```suggestion
    enum sex: { man: 0, woman: 1 }
```

Reply:
でも、enumで定義しようという考え自体はgoodです 👍